### ISSUE TITLE
Fix Multiple headings in `hgroup` element not alerted

### ISSUE THREAD

**Issue by SaekiTominaga** [markuplint/markuplint#566]
- Markuplint version: `2.11.1`
- Parser lang: HTML
- Node.js version: `18.12.0`
- OS: Windows

## Describe the bug

According to the HTML spec, the content model for the [`hgroup` element](https://html.spec.whatwg.org/multipage/sections.html#the-hgroup-element) is defined as follows:

> Zero or more p elements, followed by **one** h1, h2, h3, h4, h5, or h6 element, followed by zero or more p elements, optionally intermixed with script-supporting elements.

Therefore, the following HTML is invalid, but markuplint does not issue warnings.

```html
<hgroup>
  <h1>heading1</h1>
  <h2>heading2</h2>
</hgroup>
```
